uvtools/dspec.py

```diff
@@ -62,8 +62,8 @@ def calc_width(filter_size, real_delta, nsamples):
 
 
 def high_pass_fourier_filter(data, wgts, filter_size, real_delta, clean2d=False, tol=1e-9, window='none',
-                             skip_wgt=0.1, maxiter=100, gain=0.1, filt2d_mode='rect', alpha=0.5,
-                             edgecut_low=0, edgecut_hi=0, add_clean_residual=False):
+                             skip_wgt=0.1, maxiter=100, gain=0.1, filt2d_mode='rect', alpha=0.5, linear = False,
+                             edgecut_low=0, edgecut_hi=0, add_clean_residual=False, cache = {}):
     '''Apply a highpass fourier filter to data. Uses aipy.deconv.clean. Default is a 1D clean
```

Comment:
should update this to read uses aipy.deconv.clean if linear == False. also perhaps better to put `linear` kwarg next to `cache` in the function signature, and same fro docstring

Reply:
I've rearranged the arg order to reflect the doc string.